usrsctplib/netinet/sctp_timer.c

```diff
@@ -606,7 +606,7 @@ sctp_mark_all_for_resend(struct sctp_tcb *stcb,
 				 */
 				if (SCTP_BASE_SYSCTL(sctp_logging_level) & SCTP_FR_LOGGING_ENABLE) {
 					sctp_log_fr(0,
-						    chk->sent_rcv_time.tv_sec,
+							(uint32_t)chk->sent_rcv_time.tv_sec,
```

Comment:
Fixed in https://github.com/sctplab/usrsctp/commit/0401887b6b7679a62868f7c7d0b3d5a000b25fde.